src/components/Navbar.tsx

```diff
@@ -0,0 +1,69 @@
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { useTranslation } from "react-i18next";
+import {
+  faHouseChimney,
+  faBookSkull,
+  faHeadSideVirus,
+  faRightToBracket,
+} from "@fortawesome/free-solid-svg-icons";
+
+import { GiAngelWings } from "react-icons/gi";
+
+function Navbar() {
+  const home_icon = <FontAwesomeIcon icon={faHouseChimney} />;

```

Comment:
The convention for declaring variables that hold components is as if u create a component with pascal case - HomeIcon for example. Also here it is not actually needed to save the component in variable u can just put it inside the a tag. If the component was used more than once it may be a good way but since it is used only once it is not necessary